package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "espn-fantasy-autopick",
-    "version": "3.0.3",
-    "description": "A Chrome extension that lets you automatically add active players to the current roster in an ESPN NBA/NHL fantasy league.",
+    "version": "3.0.4",
+    "description": "An extension that lets you automatically add active players to the current roster in an ESPN NBA/NHL fantasy league.",
```

Comment:
Maybe "A browser extension that lets you..."?

Reply:
Done in 0d5ae90fa17bc30f95756a3224f4cdb75d8c48d3